docs/source/tutorials/private_network.md

```diff
@@ -4,85 +4,80 @@ All Rights Reserved
 SPDX-License-Identifier: Apache-2.0
 )
 
-## Start private Kagome network
+# Starting a Blockchain Network: A Step-by-Step Guide
 
-In this tutorial we will learn how to start a blockchain network with a validator and block producing nodes
+In this tutorial, we will guide you through the process of setting up a blockchain network with both validating and syncing nodes.
 
-First go to tutorial's folder:
+## Step 1: Navigate to the Tutorial's Directory
+
+Start by navigating to the directory containing the necessary resources for this tutorial.
 
 ```shell script
 cd examples/network
 ```
 
-### Execute first block_validator node
-
-First we execute block_validator node in the similar way we did it during previous tutorial. This node will produce and finalize blocks.
+## Step 2: Initialize the First Validating Node
 
-To start with let's navigate into the node's folder:
+In this step, we will launch the first validating node, similar to what we did in the previous tutorial. This node will be responsible for producing and finalizing blocks.
```

Comment:
(one node won't produce blocks, it will wait until at least one other node (even non validator) connects, before producing blocks)